odoo/fields.py

```diff
@@ -3485,12 +3485,14 @@ def _add_default_values(self, env, values):
         if not properties_definition:
             return properties_values
 
-        assert isinstance(properties_values, (list, dict))
+        assert isinstance(properties_values, (list, dict, str))
```

Comment:
whith `eval` can't we load properties in demo data ?
```
<field eval="[{...}]" name="properties"/>
```

Reply:
Oh ! It works ! :scream: I will just do that because it's ten times better: We will then be able to use the `ref` function in the demo data to link a record to the relational property fields and we will not have to stringify the domains.
Thank you! I will close the community PR once I updated everything.